docs/json_transformations.md

```diff
@@ -74,46 +71,50 @@ data class Example(
     @Serializable(WrappingJsonListSerializer::class) val moreData: List<StringData>
 )
 
-// And be able to parse different shapes of the same data, such as:
-```
+// And for input 
+{"name":"test","data":{"data":"str1"},"moreData": {"data":"str2"} }
```

Comment:
Why have you erased other examples?

Reply:
To reduce the amount of cognitive load, typically one example is enough